bitnami/harbor/templates/notary/notary-svc.yaml

```diff
@@ -11,12 +12,12 @@ metadata:
   annotations: {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
   {{- end }}
 spec:
-{{- if (eq .Values.ingress.controller "gce") }}
+  {{- if (eq .Values.ingress.core.controller "gce") }}
   type: NodePort
-{{- end }}
+  {{- end }}
   ports:
     - name: notary-server
-      port: 4443
+      port: {{ .Values.notary.server.containerPorts.server }}
```

Comment:
This should be service port, not container port

Reply:
To be resolved in previous conversation.